src/PhysicsEngine.cpp

```diff
@@ -277,6 +280,9 @@ void RungeKutta::march_step(double tStamp, double tStep) {
     rocket_.set_f_net(net_force_if);
     rocket_.set_t_net(net_torque_if);
     rocket_.set_q_ornt(orient);
+
+    rocket_.set_r_ecef(i2ecef(pos_if));
+    rocket_.set_r_geod(ecef2geod(ecef));
```

Comment:
you can't call these setters with an r-value, changing the set_r_...(Vector3d & vect) functions to set_r_...(Vector3d vect) will fix this